product-service/handlers/products/addProduct/addProduct.ts

```diff
@@ -0,0 +1,18 @@
+import 'source-map-support/register';
+import {buildResponse, ErrorCode} from "../../common/helpers";
+import {APIGatewayProxyHandler} from "aws-lambda";
+import {productDao} from "../../../dao/product/productDao";
+import {emptyProduct, Product} from "../../../dao/daoAPI";
+import {ProductNotFound} from "../../../errors/errors";
+
+export const addProductHandler: APIGatewayProxyHandler = async ({body}) => {
+    try {
+        const requestData: Product = JSON.parse(body) as Product;
+        const [result] = await productDao.add({...emptyProduct, ...requestData}) as Product[]
+        return buildResponse(result);
+    } catch (error) {
+        return error instanceof ProductNotFound
+            ? buildResponse(error.message, ErrorCode.BAD_REQUEST)
```

Comment:
not validate all field 
![image](https://user-images.githubusercontent.com/72539820/98648987-aa2e3980-233f-11eb-9721-ac9d0ecf0a4f.png)